src/pages/staff/dashboard/MyBrandsPage.jsx

```
@@ -62,13 +64,23 @@ export default function MyBrandsPage() {
   }, [isDesktop]);
   
   // Search state
+  const STORAGE_KEY = 'en.search.myBrands';
   const [searchQuery, setSearchQuery] = useState('');
   const [allBrands, setAllBrands] = useState([]);
   const [displayBrands, setDisplayBrands] = useState([]);
   const [loading, setLoading] = useState(true);
   const [selectedBrandId, setSelectedBrandId] = useState(null);
```

ESLint

[error] 'selectedBrandId' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)